.github/workflows/inductor-perf-test-nightly-x86.yml

```diff
@@ -90,15 +98,14 @@ jobs:
       selected-test-configs: ${{ inputs.benchmark_configs }}
     secrets: inherit
 
-
-  linux-jammy-cpu-py3_9-gcc11-inductor-test-nightly:
+  linux-jammy-cpu-py3_9-gcc11-inductor-test-nightly-freezing:
     name: linux-jammy-cpu-py3.9-gcc11-inductor
     uses: ./.github/workflows/_linux-test.yml
     needs: linux-jammy-cpu-py3_9-gcc11-inductor-build
-    if: github.event.schedule == '0 7 * * *'
+    if: github.event.schedule == '0 7 * * *' || github.event_name == 'pull_request'
```

Comment:
I assume that you will remove `|| github.event_name == 'pull_request'` part before landing this?

Reply:
```suggestion
    if: github.event.schedule == '0 7 * * *'
```